DeductoActivity.py

```diff
@@ -44,21 +37,13 @@
 _logger = logging.getLogger('deducto-activity')
 
 
-SERVICE = 'in.seeta.Deducto'
-IFACE = SERVICE
-PATH = '/in/seeta/DeductoActivity'
-
-
-
 class DeductoActivity(activity.Activity):
     """ Logic puzzle game """
 
     def __init__(self, handle):
         """ Initialize the toolbars and the game board """
-        try:
-            super(DeductoActivity, self).__init__(handle)
-        except dbus.exceptions.DBusException as e:
-            _logger.error(str(e))
+
+        super(DeductoActivity, self).__init__(handle)
```

Comment:
Am I right, if `super(DeductoActivity, self).__init__(handle)` can be simplified to `super().__init__(handle)` ?

Reply:
Yeah you are , these arguments were necessary in Python 2 , but in Python 3  we can use either of them. Thanks